Mathlib/Data/DFinsupp/Basic.lean

```diff
@@ -1522,7 +1522,6 @@ theorem sigmaCurry_single [∀ i, DecidableEq (α i)] [∀ i j, Zero (δ i j)]
     simp [hi]
 #align dfinsupp.sigma_curry_single DFinsupp.sigmaCurry_single
 
-/- ./././Mathport/Syntax/Translate/Expr.lean:107:6: warning: expanding binder group (i j) -/
```

Comment:
Eg this one is still applicable since there is `Π₀ (i) (j), δ i j` just a few lines under.